.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Instead of doing this, I'd like to suggest to keep a personal global gitignore file: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Done.